addons/account/migrations/9.0.1.1/pre-migration.py

```diff
@@ -310,6 +310,14 @@ def merge_supplier_invoice_refs(env):
     )
 
 
+def set_date_maturity(cr):
+    cr.execute("""
```

Comment:
Please use `openupgrade.logged_query` for logging the operation and know how much rows this affects